src/MongoDB.Analyzer.Helpers/Builders/MqlGenerator.cs

```diff
@@ -41,11 +43,81 @@ private sealed class MqlGeneratorTemplateType
         public static string GetDriverVersion() => typeof(Builders<>).Assembly.GetName().Version.ToString(3);
         public static int[] LinqReference => Enumerable.Range(1, 10).ToArray();
 
+        private static string Render<T, E>(object buildersDefinition)
```

Comment:
The order of methods:
public,
internal,
private

Reply:
Resolved